tests/test_php_semantic.py

```
@@ -0,0 +1,383 @@
+        assert "complexity:" in classify.cfg_summary
+        assert classify.dfg_summary != "", f"classify should have dfg_summary, got: {classify.dfg_summary}"
+        assert "vars:" in classify.dfg_summary
+
+        # Check method extraction
+        method_units = [u for u in units if u.unit_type == "method"]
+        assert len(method_units) >= 1, f"Should extract at least 1 method, got {len(method_units)}"
+
+        getUser = next((u for u in method_units if u.name == "getUser"), None)
+        assert getUser is not None, "getUser method should be extracted"
+        # Check CFG summary for method
+        assert getUser.cfg_summary != "", f"getUser should have cfg_summary"
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)